src/utils/helpers/loginCheck.ts

```diff
@@ -1,7 +1,7 @@
 const isLoggedIn = () => {
   const JWT = localStorage.getItem('persistJWT')
-  const username = localStorage.getItem('username')
-  return JWT && username
+  const user = localStorage.getItem('user')
+  return JWT && user
 }
```

Comment:
Maybe here should also update the user in the context? I haven't tested yet, but I'm assuming reloading would fail?

Reply:
Yes, I was going to make the context update go here, but it's not a React component, so can't use Hooks, etc.

See my above comment for how it's being handled now.